index.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter&display=swap"
+      rel="stylesheet"
+    />
+    <title>Document</title>
+  </head>
+  <body>
+    <header class="nav-section">
+      <div class="welcome-heading">
+        <h1>Welcome</h1>
+      </div>
+      <nav>
+        <img class="menu" src="assets/images/Icon-Menu.png" alt="Menu icon" />
+      </nav>
+    </header>
+    <section class="first-section">
+      <div class="flex-box">
+        <br />
+        <br />
+        <br />
+        <div class="flex-design">
```

Comment:
- [ ] Instead of line breaks `br`, I suggest using CSS ` margin-top` from the `flex-design` class or `padding-top` from the `flex-box` class

Reply:
Great work implementing the required changes. 👍🏼 @faizi2500